pkg/controller/controller.go

```diff
@@ -157,25 +158,26 @@ func (c *Controller) processPodItem(pod *v1.Pod) error {
 		}
 	}
 	if len(pvcKey) == 0 {
-		// For some reason, no pvc matching the volume name was found.
-		return fmt.Errorf("processPodItem: Pod does not contain volume %q", DataVolName)
+		// If this block is ever reached, something has gone very wrong.  The pod should ALWAYS be created with the volume.
+		// A missing volume would most likely indicate a pod that has been created manually, but also incorrectly defined.
+		return errors.Errorf("Pod does not contain volume %q", DataVolName)
 	}
 	glog.Infof("processPodItem: Getting PVC object for key %q", pvcKey)
 	pvc, err := c.pvcFromKey(pvcKey)
 	if err != nil {
-		return fmt.Errorf("processPodItem: error getting pvc from key: %v", err)
+		return errors.Wrapf(err, "error getting pvc object with key %q", pvcKey)
 	}
 	err = c.setPVCAnnotation(pvc, AnnPodPhase, string(pod.Status.Phase))
 	if err != nil {
-		return fmt.Errorf("processPodItem: error setting PVC annotation: %v", err)
+		return errors.Wrapf(err, "error setting annotation %q:%q in pvc %q", AnnPodPhase, pod.Status.Phase, pvc.Name)
```

Comment:
here we call `errors.Wrapf` but on L241, after also calling setPVCAnnotation, we call `errors.WithMessage`. Why the difference?